package.json

```diff
@@ -24,10 +24,11 @@
     "lint:fix": "npm run lint -- --fix",
     "lint:fix-cached": "npm run lint:fix -- --cache",
     "postinstall": "npx lerna bootstrap --force-local --strict",
-    "check:license": "npx license-checker -direct --excludePackages \"@sap/edm-converters@1.0.21;@sap/xsenv@2.2.0;weak-map@1.0.5\" --onlyAllow \"MIT;Apache;ISC;BSD;WTFPL;Artistic;CC-BY;CC0;Unlicense;Public Domain\"",
+    "check:license": "npx license-checker -direct --excludePackages \"@sap/edm-converters@1.0.21;@sap/xsenv@2.2.0;weak-map@1.0.5\" --onlyAllow \"MIT;Apache;ISC;BSD;WTFPL;Artistic;CC-BY;CC0;Unlicense;Public Domain;GPL-3.0\"",
     "check:vulnerability": "npx audit-ci -h",
     "bump:patch": "npx lerna version patch --force-publish -y",
-    "bump:minor": "npx lerna version minor --force-publish -y"
+    "bump:minor": "npx lerna version minor --force-publish -y",
+    "check:circular": "madge --circular ./packages/core"
```

Comment:
So, this is just a useful tool that is optional for the current status.
Let's check it in the pipeline in the follow up.

Reply:
Yes, unfortunately there is no build in threshold option. So we either fix them all or build it by ourself or make an OS contribution to madge.